### PR TITLE
Minor improvements to issue template

### PR DESCRIPTION
**Pull request by nyuszika7h**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

* Updated the issue template to collapse logs with HTML5 `<details>` and `<summary>`. This may make it easier to scroll through issues with long logs.
* Moved the command line input part from the logs to above, since it doesn't need to be included as it will be output with `-v` anyway, and it may contain sensitive information that is automatically redacted in the logs (hopefully people are smart enough to redact any sensitive information, but who knows).
* Made other minor grammar/formatting corrections.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE_tmpl.md [hunk 1]
```diff
@@ -28,9 +28,12 @@
 
 ### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:
 
-Add `-v` flag to **your command line** you run youtube-dl with, copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):
+Add `-v` flag to **your command line** you run youtube-dl with (`youtube-dl -v <your command line>`), copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):
+
+<details>
+<summary>Log</summary>
```
Comment (dstftw) [inline .github/ISSUE_TEMPLATE_tmpl.md]: Verbose output is the central part of the report and it should not be collapsed by default. As a developer I don't want to take any additional actions to make it visible.

Comment (nyuszika7h) [inline .github/ISSUE_TEMPLATE_tmpl.md]: Okay, removed this change.

File: .github/ISSUE_TEMPLATE_tmpl.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
 ## Please follow the guide below
 
-- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
-- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
-- Use *Preview* tab to see how your issue will actually look like
+- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly.
+- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`).
+- Use the *Preview* tab to see what your issue will actually look like.
```
Comment (dstftw) [inline .github/ISSUE_TEMPLATE_tmpl.md]: What's the point of trailing dots? They are not used in the rest of template for lists' items.

Comment (nyuszika7h) [inline .github/ISSUE_TEMPLATE_tmpl.md]: They are complete sentences unlike the others. Though to be fair, the license ones should have trailing dots too. But I can remove them if you want.

Comment (nyuszika7h) [inline .github/ISSUE_TEMPLATE_tmpl.md]: So, what should I do then?

Comment (dstftw) [inline .github/ISSUE_TEMPLATE_tmpl.md]: I've requested to remove them.

Comment (nyuszika7h) [inline .github/ISSUE_TEMPLATE_tmpl.md]: Done, sorry for the delay.

**Pull request merged**